### PR TITLE
Added that button disables with no username

### PR DESCRIPTION
**Pull request by jugendhacker**
Solved #1

### CODE REVIEW

File: src/main/resources/static/app.js [hunk 1]
```diff
@@ -74,5 +74,12 @@ $(function () {
         })
     }), $("#send").click(function () {
         sendMessage()
+    }), $("#name").on('keyup', function () {
```
Comment (amarlearning) [inline src/main/resources/static/app.js]: If the user simply clicks on the `connect to` button without touching the `username` field then it's allowing.

Comment (jugendhacker) [reply src/main/resources/static/app.js]: Oh I will change it...

Comment (jugendhacker) [reply src/main/resources/static/app.js]: I think this is solved with 52924a4176f38b8ccac81706c5ecc0241a882858

File: src/main/resources/static/main.css [hunk 2]
```diff
@@ -82,6 +82,10 @@ i{
 .toggle>.toggle-group>.toggle-off,.toggle>.toggle-group>.toggle-on{
     padding-left:25px
 }
+.toggle[disabled]{
```
Comment (amarlearning) [inline src/main/resources/static/main.css]: let's change the `cursor` from `pointer` to `not-allowed`

Comment (jugendhacker) [reply src/main/resources/static/main.css]: Why?

Comment (amarlearning) [reply src/main/resources/static/main.css]: @jugendhacker And let's change the `background-color` to `#AAAAAA`

Comment (amarlearning) [reply src/main/resources/static/main.css]: while the button is disabled there is no point of keeping `cursor` as `pointer`

Comment (jugendhacker) [reply src/main/resources/static/main.css]: Done

**Comment by jugendhacker** [comment]
What is the problem with the formatting?

**Comment by amarlearning** [comment]
@jugendhacker code is not formatted properly :sweat_smile:

**Comment by jugendhacker** [comment]
Please comment the lines where you have a problem with the formatting of the code, because for me there is no formatting problem...

File: src/main/resources/static/app.js [hunk 3]
```diff
@@ -74,5 +74,12 @@ $(function () {
         })
     }), $("#send").click(function () {
         sendMessage()
-    })
-});
\ No newline at end of file
+    }), $("#name").on('keyup', function () {
+        var object = $(this).val();
+        if (object.length <= 0){
```
Comment (amarlearning) [inline src/main/resources/static/app.js]: this

File: src/main/resources/static/app.js [hunk 4]
```diff
@@ -74,5 +74,12 @@ $(function () {
...
+            $("#toggle-event").bootstrapToggle('disable');
+        }else {
```
Comment (amarlearning) [inline src/main/resources/static/app.js]: this

**Comment by jugendhacker** [comment]
Sorry but I don't see your problem there, you don't want to explain and you have no public formatting guidelines so how should I know how you want the format?

File: src/main/resources/static/app.js (hunk 3 above)
Comment (amarlearning) [reply src/main/resources/static/app.js]: can you add a space before parenthesis?

File: src/main/resources/static/app.js (hunk 4 above)
Comment (amarlearning) [reply src/main/resources/static/app.js]: can you add a space before `else` keyword?

**Comment by jugendhacker** [comment]
Resolved it

**Pull request merged**